HW03/Darya_Zhalova/darya_zhalova_hw_03_t_03.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+class Homework3
+  REG_TIME = /\d{2}:\d{2}\.\d/
+
+  def task3(log)
+    return '0' unless if_log_valid(log)
+
+    times = make_array_time(log)
+    arr = []
+    (times.size - 1).times do |i|
+      arr.append (times[i + 1] - times[i]).abs.round(1).to_s
+    end
+    return arr.first if arr.size == 1
+
+    arr
+  end
+
+  def if_log_valid(log)
+    log.sub('Calling core with action:', '').include? 'Calling core with action:'
+  end
+
+  def make_array_time(log)
```

Comment:
No need to use `verb` if you just want to return something
```suggestion
  def array_time(log)
```